velox/dwio/common/tests/utils/FilterGenerator.cpp

```diff
@@ -91,7 +92,8 @@ std::unique_ptr<Filter> ColumnStats<bool>::makeRangeFilter(
 template <>
 std::unique_ptr<Filter> ColumnStats<UnscaledShortDecimal>::makeRangeFilter(
     float startPct,
-    float selectPct) {
+    float selectPct,
+    FilterSpec filterSpec) {
```

Comment:
FilterSpec contains startPct and selectPct already. It might be better to create two set of signatures
```
template <>
std::unique_ptr<Filter> ColumnStats<>::makeRangeFilter(FilterSpec filterSpec) {
...
}

template <typename T>
std::unique_ptr<Filter> ColumnStats<>::makeRangeFilter(float startPct, float selectPct) {
    FilterSpec filterSpec(startPct, selectPct);
    return makeRangeFilter<T>(filterSpec);
}
```